auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -189,40 +195,27 @@ public static int getDefaultTheme() {
     /**
      * Signs the current user out, if one is signed in.
      *
-     * @param activity the activity requesting the user be signed out
+     * @param context the context requesting the user be signed out
      * @return A task which, upon completion, signals that the user has been signed out ({@link
      * Task#isSuccessful()}, or that the sign-out attempt failed unexpectedly !{@link
      * Task#isSuccessful()}).
      */
     @NonNull
-    public Task<Void> signOut(@NonNull FragmentActivity activity) {
-        // Get Credentials Helper
-        GoogleSignInHelper signInHelper = GoogleSignInHelper.getInstance(activity);
-
+    public Task<Void> signOut(@NonNull Context context) {
         // Firebase Sign out
         mAuth.signOut();
 
         // Disable credentials auto sign-in
-        Task<Status> disableCredentialsTask = signInHelper.disableAutoSignIn();
-
-        // Google sign out
-        Task<Status> signOutTask = signInHelper.signOut();
-
-        // Facebook sign out
-        try {
-            LoginManager.getInstance().logOut();
-        } catch (NoClassDefFoundError e) {
-            // do nothing
+        CredentialsClient client;
+        if (context instanceof Activity) {
+            client = Credentials.getClient((Activity) context);
+        } else {
+            client = Credentials.getClient(context);
         }
```

Comment:
I looked at the PS code and it doesn't do this check for us which is too bad. We want to make sure to show a `Fix this Play Services problem` dialog if possible instead of just showing a notification.